web/src/user_sort.ts

```diff
@@ -24,13 +26,13 @@ export function sort_email(a, b) {
         // User with hidden should be at last.
         return -1;
     }
-    return compare_a_b(email_a.toLowerCase(), email_b.toLowerCase());
+    return compare_a_b(email_a?.toLowerCase(), email_b?.toLowerCase());
```

Comment:
```suggestion
    return compare_a_b(email_a.toLowerCase(), email_b.toLowerCase());
```

Reply:
@andersk I have made the required changes. Please take a look.